conformity/fields/meta.py

```diff
@@ -162,7 +184,7 @@ def errors(self, value):
             ok = self.validator(value)
         except Exception:
             return [
-                Error("Validator errored (invalid type?)"),
+                Error("Validator erred (invalid type?)"),
```

Comment:
If we're going deeply grammatical, then it's incorrect. Validator didn't _err_, it encountered an _error_.

Reply:
Changed.